docs/guides/tutorials/chatgpt_bot.rst

```diff
@@ -0,0 +1,2305 @@
+          "jsx": "preserve",
+          "incremental": true,
+          "plugins": [
+            {
+              "name": "next"
+            }
+          ],
+          "paths": {
+            "@/*": ["./*"]
+    -     }
+    +     },
+    +     "baseUrl": "."
+        },
+        "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+        "exclude": ["node_modules"]
+      }
+
+.. lint-on
+
+Now, we'll create an instance of EdgeDB for our project, but first, we need to
+install EdgeDB!
+
+
+Install the EdgeDB CLI
+----------------------
+
+*If you already have EdgeDB installed, you can skip to creating an instance.*
+
+Before we can create an instance for our project, we need to install the EdgeDB
+CLI. On Linux or MacOS, run the following in your terminal and follow the
+on-screen instructions:
+
+.. code-block:: bash
+
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
+
+Windows Powershell users can use this command:
+
+.. code-block:: powershell
+
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+For other installation scenarios, see the "Additional installation methods"
+section of `our "Install" page <https://www.edgedb.com/install>`_.
+
+
+Create a local EdgeDB instance
+------------------------------
+
+To create our instance, let's initialize our project as an EdgeDB project. Run
+the following in the root of the project:
+
+.. code-block:: bash
+
+    $ edgedb project init
+    No `edgedb.toml` found in `/<path>/<to>/<project>/docs-chatbot`
+    or above
+
+    Do you want to initialize a new project? [Y/n]
+    > Y
+
+    Specify the name of EdgeDB instance to use with this project
+    [default: docs_chatbot]:
+    > docs_chatbot
+
+    Checking EdgeDB versions...
+    Specify the version of EdgeDB to use with this project
+    [default: 3.2]:
+    > 3.2
+
+The CLI should set up an EdgeDB project, an instance, and a database within
+that instance.
+
+- Confirm project creation by checking for an ``edgedb.toml`` file and a
+  ``dbschema`` directory in the project root.
+- Confirm the instance is running with the ``edgedb instance list`` command.
+  Search for the name of the instance you've just created (``docs_chatbot`` if
+  you're following along) and check the status. (Don't worry if the status is
+  "inactive"; the status will change to "running" automatically when you
+  connect to the instance.)
+- Confirm you can connect to the created instance by running ``edgedb`` in the
+  terminal to connect to it via REPL or by running ``edgedb ui`` to connect
+  using the UI.
+
+
+Configure the environment
+-------------------------
+
+Create a ``.env.local`` file in the root of your new Next.js project.
+
+.. code-block:: bash
+
+    $ touch .env.local
+
+We're going to add a couple of variables to that file to configure EdgeDB.
+We'll need to run a command on our new instance to get the value for one of
+those. Since the runtime we'll be using in our Next.js project does not have
```

Comment:
I think we can explain this a bit better. @jaclarke helped me to come with this:

`We're going to add a couple of variables to that file to configure the EdgeDB client. We'll need to run a command on our new instance to get the value for one of those. Since we'll be using the [Edge Runtime](https://nextjs.org/docs/app/api-reference/edge) in our Next.js project, the edgedb-js client won't be able to access the Node.js filesystem API's it usually uses to automatically find your instance, so we need to provide the DSN for the instance instead. To get that, run this command:`